--- raw_ui_mainwindow.py	2021-06-23 06:19:29.874024478 +0800
+++ ui_mainwindow.py	2021-06-23 06:19:32.860652463 +0800
@@ -9,10 +9,11 @@
 
 
 from PyQt5 import QtCore, QtGui, QtWidgets
+from PyQt5.QtWidgets import QMainWindow
 
 
 class Ui_MainWindow(object):
-    def setupUi(self, MainWindow):
+    def setupUi(self, MainWindow: QMainWindow) -> None:
         MainWindow.setObjectName("MainWindow")
         MainWindow.resize(844, 634)
         self.centralwidget = QtWidgets.QWidget(MainWindow)
@@ -179,7 +180,7 @@
         self.retranslateUi(MainWindow)
         QtCore.QMetaObject.connectSlotsByName(MainWindow)
 
-    def retranslateUi(self, MainWindow):
+    def retranslateUi(self, MainWindow: QMainWindow) -> None:
         _translate = QtCore.QCoreApplication.translate
         MainWindow.setWindowTitle(_translate("MainWindow", "渲染大作业 By Sun"))
         self.dockScene.setWindowTitle(_translate("MainWindow", "对象树"))
